src/NUnitFramework/tests/Internal/PlatformDetectionTests.cs

```diff
@@ -383,37 +383,13 @@ public void DetectMonoTouch()
         }
 
         [Test]
-        public void DetectNetCore20()
+        public void DetectNetCore()
         {
             CheckRuntimePlatforms(
-                new RuntimeFramework(RuntimeType.NetCore, new Version(2, 0, 0)),
-                "NetCore,NetCore-2.0");
+                new RuntimeFramework(RuntimeType.NetCore, new Version(0, 0, 0)),
+                "NetCore");
         }
-
-        [Test]
-        public void DetectNetCore21()
-        {
-            CheckRuntimePlatforms(
-                new RuntimeFramework(RuntimeType.NetCore, new Version(2, 1, 0)),
-                "NetCore,NetCore-2.0,NetCore-2.1");
```

Comment:
Would you please add a test verifying that `NetCore-2.0` errors, rather than skipping or matching?

Reply:
I would expect PlatformHelper to throw an InvalidPlatformException, which will be caught by the platformattribute and mark the test not-runnable. There's an example of that for another case - it should be possible just to extend that to cover this case too.

That means that a user will get clear feedback if they try and create a test with the NetCore-2.0, when it is not valid.

Reply:
Sounds good, I'll make that happen.